js_modules/dagit/packages/core/src/runs/RunTable.tsx

```diff
@@ -244,7 +244,7 @@ const RunRow: React.FC<{
   };
 
   return (
-    <Row key={run.runId} highlighted={!!isHighlighted}>
```

Comment:
Removed this while hunting for a key error. It's not needed, so I left it removed.